CHANGELOG.md

```diff
@@ -176,6 +168,20 @@ been added to each release, please refer to the [blog](https://blog.gitea.io).
   * Fix wrong original git service type on a migrated repository (#9693)
   * Fix ref links in issue overviews for tags (#8742)
 * ENHANCEMENTS
+  * Fix search form button overlap (#11840) (#11864)
+  * Make tabular menu styling consistent for arc-green (#11570) (#11798)
+  * Add option to API to update PullRequest base branch (#11666) (#11796)
+  * Increase maximum SQLite variables count to 32766 (#11696) (#11783)
+  * Update emoji dataset with skin tone variants (#11678) (#11763)
+  * Add logging to long migrations (#11647) (#11691)
+  * Change language statistics to save size instead of percentage (#11681) (#11690)
```

Comment:
```suggestion
  * Change language statistics to save size instead of percentage (#11681) (#11690)
  * Allow different HardBreaks settings for documents and comments (#11515) (#11599)
```